ee/localserver/request-controlservice.go

```diff
@@ -5,37 +5,54 @@ import (
 	"fmt"
 	"net/http"
 	"time"
+
+	"github.com/kolide/launcher/pkg/traces"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
 )
 
 func (ls *localServer) requestAccelerateControlHandler() http.Handler {
 	return http.HandlerFunc(ls.requestAccelerateControlFunc)
 }
 
 func (ls *localServer) requestAccelerateControlFunc(w http.ResponseWriter, r *http.Request) {
+	_, span := traces.New(r.Context(), trace.WithAttributes(attribute.String(traces.AttributeName("localserver", "path"), r.URL.Path)))
+	defer span.End()
+
 	if r.Body == nil {
+		span.SetStatus(codes.Error, "request body is nil")
 		sendClientError(w, "request body is nil")
```

Comment:
Why not pass the span to `sendClientError`?

(Or make a wrapper, that pulls errors out of request context)